packages/MSBot/src/msbot-connect-file.ts

```diff
@@ -2,14 +2,15 @@
  * Copyright(c) Microsoft Corporation.All rights reserved.
  * Licensed under the MIT License.
  */
-import * as chalk from 'chalk';
+// tslint:disable:no-console
+ import * as chalk from 'chalk';
 import * as program from 'commander';
 import * as path from 'path';
 import { BotConfig } from './BotConfig';
 import { FileService } from './models/fileService';
 import { IFileService, ServiceType } from './schema';
 
-program.Command.prototype.unknownOption = function (flag: any) {
+program.Command.prototype.unknownOption = function (flag: any): void {
```

Comment:
Remove changes unrelated to the no-console issue so they are more easily reviewed.